pkg/assetfetcher/http.go

```diff
@@ -88,6 +88,10 @@ func (c *NexposeAssetFetcher) FetchAssets(ctx context.Context, siteID string) (<
 	pagedErrChan := make(chan error, siteAssetResp.Page.TotalResources)
 
 	for _, asset := range siteAssetResp.Resources {
+		if !asset.hasBeenScanned() {
+			// no need to continue processing a Nexpose Asset that has never been scanned
+			continue
```

Comment:
Suggestion: Could you add a log and/or custom metric here so we have a sense of how many unscanned assets we're skipping? When we start to build out services to configure the scan sites in Nexpose, I could see this sort of information would very helpful.

Reply:
I really like the custom metric idea.  I'll add it.